modules/account-lib/src/coin/baseCoin/enum.ts

```diff
@@ -3,6 +3,8 @@
  */
 export enum TransactionType {
   Send,
+  SendERC721,
```

Comment:
Could we get matching Initialization comments here to match the rest of the comments in this enum?